doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -65,18 +65,18 @@ endif::[]
 endif::offline_onprem[]
 . On the computer that has an outgoing connection to the public Internet, complete the following steps:
 .. Download the latest version of the http://release-engineering.github.io/offliner/[Offliner tool].
-.. Download the `{PRODUCT_FILE}-offliner.txt` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
-.. Enter the following command to use the Offliner tool to download the required artifacts:
+.. Download the `{PRODUCT_FILE}-offliner.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
+.. Extract the contents of the `{PRODUCT_FILE}-offliner.zip` file into any directory.
+.. Change to the directory and enter the command:
 +
 [subs="attributes,verbatim,macros"]
 ----
-java -jar offliner-<version>.jar -r \https://maven.repository.redhat.com/ga/ -r \https://repo1.maven.org/maven2/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
+./offline-repo-builder.sh offliner.txt 
 ----
 +
-Replace `/home/user/temp` with an empty temporary directory and `<version>` with the version of the Offliner tool that you downloaded. The download can take a significant amount of time.
-
+This command creates a `repository` subdirectory and downloads the necessary artifacts into this subdirectory. Ignore the message that some downloads have failed. 
```

Comment:
Recent offliner manifest tests (for RHPAM/RHDM 7.7.0) report zero failed downloads. I am wondering if we could remove the suggestion to ignore failed downloads?